pandas/core/generic.py

```diff
@@ -6703,13 +6703,39 @@ def at_time(self, time, asof=False):
         """
         Select values at particular time of day (e.g. 9:30AM).
 
+        Notes
+        -----
+        For this method to work, the index must to be a :class:`DateTimeIndex`
+
         Parameters
         ----------
         time : datetime.time or string
 
         Returns
         -------
         values_at_time : type of caller
+
+        Examples
+        --------
+        >>> i = pd.date_range('2018-04-09', periods=4, freq='4D2min')
+        >>> ts = pd.DataFrame({'A': [1,2,3,4]}, index=i)
+        >>> ts
+                             A
+        date
+        2018-04-09 00:00:00  1
+        2018-04-13 00:02:00  2
+        2018-04-17 00:04:00  3
+        2018-04-21 00:06:00  4
+        >>> ts.at_time('0:02')
+                             A
+        date
+        2018-04-13 00:02:00  2
+
+        See Also
+        --------
+        between_time : Select values between particular times of the day
+        first : select initial periods of time series based on a date offset
+        last: select final periods of time series based on a date offset
```

Comment:
The capitalization is inconsistent at the beginning of the descriptions: "S" vs "s" in "select".

Reply:
Likewise in the other "See Also" sections